tests/mir-opt/unnamed-fields/field_access.rs

```diff
@@ -1,4 +1,5 @@
-// skip-filecheck
+//@ unit-test: UnnamedFields
```

Comment:
There is no mir pass with such a name. Could you prefer a plain text comment ?

Reply:
I have updated the pass name to `// Tests the correct handling of unnamed fields within structs and unions marked with #[repr(C)]` text comment.